site/content/contribute/getting-started/code-review.md

```diff
@@ -99,11 +117,10 @@ If you are a core committer asked to give a review
     * Don't be afraid to ask for changes repeatedly until all concerns are addressed.
     * Feel free to challenge assumptions and timelines. Rushing a change into a patch release may cause more harm than good.
 4. Merge the pull request.
-    * Do not merge until the reviewing product manager has approved the changes and removed the `1: PM Review` label.
-    * Do not merge until the reviewing QA tester has approved the changes and added the `QA Review Done` label.
-    * Do not merge until at least two core committers have approved and all concerns have been addressed.
-    * Remove any remaining `2: Dev Review` label and assign the `3: Reviews Complete` label.
+    * Do not merge until the `1: PM Review`, `2: Dev Review` and `3: QA Review` labels have been removed.
+    * Do not merge if there are outstanding changes requested.
+    * Do not merge if there are any `Do Not Merge` labels applied.
+        - When in doubt, leave the merging of the pull request to the author.
```

Comment:
Since this document is from the perspective of the author ("If you are a community member seeking a review"), maybe rephrase?

Reply:
This phrasing in actually under the `If you are a core committer asked to give a review` section -- do you think the language `author` makes sense here?

Reply:
My bad, I missed the header. In this context, "author" makes sense to me. - "community member" (seeking a review) would be prob. too verbose(?).

Reply:
I /think/ author suffices here, since it might be a community member or another core committer.